src/core_plugins/kibana/server/routes/api/suggestions/register_value_suggestions.js

```diff
@@ -0,0 +1,31 @@
+import handleESError from '../../../lib/handle_es_error';
+
+export function registerValueSuggestions(server) {
+  server.route({
```

Comment:
Some [API tests](https://github.com/elastic/kibana/tree/master/test/api_integration) would be good for this route.